content/rc/subscriptions/maintenance/_index.md

```diff
@@ -26,15 +26,32 @@ This includes, but is not limited to:
 - Adding more memory to a node
 - Applying security patches
 
-Redis will notify users by email when maintenance starts and ends. If Redis needs an action from a user to start maintenance, Redis will notify users with a reasonable amount of time before planned maintenance.
-
-For major upgrades or upgrades that might include breaking changes, users will receive an advance notification with sufficient time to prepare before the upgrade.
+Redis will notify users by email when maintenance starts and ends. For more details, see [Notifications](#notifications).
 
-If you want to receive notifications by email, make sure **Operational emails** are activated in your user settings in [Access Management]({{<relref "/rc/security/access-control/access-management">}}). 
+During maintenance, your database will be operational, but you may notice some latency when connecting to your databases. Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
```

Comment:
@cmilesb, maybe we should bold it in some way and not just embed it in the paragraph

Reply:
@NoamSternRedis Bolding or italicizing entire sections of text doesn't fit our style. I separated the "disconnection" part from the "latency" part into two separate paragraphs. Do you think that's enough, or should I use a note to bring attention to the disconnection? Or, is there a specific part of this section you think needs to be bolded?